interfaces/udev/spec.go

```diff
@@ -56,6 +70,8 @@ func (spec *Specification) AddConnectedPlug(iface interfaces.Interface, plug *in
 		UDevConnectedPlug(spec *Specification, plug *interfaces.Plug, plugAttrs map[string]interface{}, slot *interfaces.Slot, slotAttrs map[string]interface{}) error
 	}
 	if iface, ok := iface.(definer); ok {
+		spec.securityTags = plug.SecurityTags()
+		defer func() { spec.securityTags = nil }()
```

Comment:
Why are we doing this? This feels strange (and dangerous), I mean, if its something that should only be valid during the iface.UDevConnectedPlug() call maybe it should be an input parameter there instead?  Actually we already pass "plug" in there, so why is UdevConntecedPlug not just using plug.SecurityTags() ?

Reply:
We do this in all the other backends. This is just making the API changes smaller and consistent with other backends.

Reply:
I agree, it merits a rethink at some point, but yes, I saw the other backends work like this as well